kubectl-rook-ceph.sh

```diff
@@ -423,15 +617,13 @@ function run_start_debug() {
   [[ -z "${REMAINING_ARGS[0]:-""}" ]] && fail_error "Missing mon or osd deployment name"
   deployment_name="${REMAINING_ARGS[0]}"              # get deployment name
   REMAINING_ARGS=("${REMAINING_ARGS[@]:1}")           # remove deploy name from remaining args
-  parse_flags parse_image_flag "${REMAINING_ARGS[@]}" # parse flags after the deployment name
-  end_of_command_parsing "${REMAINING_ARGS[@]}"
```

Comment:
@parth-gr @subhamkrai  These two lines to `parse_flags` and `end_of_command_parsing` cause an error if run when the restore-quorum command is run. But if removed like this, they don't cause a problem for the debug start command or the restore-quorum command. Am I missing anything by removing these two lines?

Reply:
`parse_flags` is to read flag args and `end_of_command_parsing` is to check if no other args is passed.

@parth-gr can give more details here.

Reply:
`end_of_command_parsing` can be removed as it just says there should not be any args after it, but which can be for this case.

parse_flags parse_image_flag "${REMAINING_ARGS[@]}" # parse flags after the deployment name
This would check if there is any alternate_image flag after deployment name,
What is the error which come if this line is kept?

Reply:
If I keep that line to parse the flags:
```
parse_flags parse_image_flag "${REMAINING_ARGS[@]}"
```

I get the error:
```
./kubectl-rook-ceph.sh: line 634: REMAINING_ARGS[@]: unbound variable
```

I'll take a closer look to make sure the `alternate_image` is still supported.